apps/obs-web/src/shims-vue.d.ts

```diff
@@ -14,3 +14,7 @@ declare module 'vue-async-function';
 declare module 'vue-multiselect';
 declare module 'vue2-timepicker';
 declare module 'vue-timeselector';
+// https://stackoverflow.com/questions/50576746/import-pdfmake-js-file-to-my-ts-file
+declare module 'pdfmake/build/pdfmake.js';
```

Comment:
is this for some future pdf stuff? If not, can probably remove.

Reply:
you should be seeing 'Export PDF' button + functionality on the DebrieferAssessment page - make sure you're looking at all commits for the pull request.

Reply:
oh nice I see it now, pretty cool feature. The pdf is formatted very nicely.